test/test_torch.py

```diff
@@ -17572,7 +17574,9 @@ def test_hypot(self, device, dtype):
                 expected = np.hypot(input[0].cpu().numpy(), input[1].cpu().numpy())
             self.assertEqual(actual, expected)
 
-    def _helper_test_igamma(self, loglo, loghi, device, dtype):
+    def _helper_test_igamma(self, loglo, loghi, device, dtype, igamma=True):
+        torch_fcn = torch.igamma if igamma else torch.igammac
```

Comment:
Instead of an argument like boolean igamma just pass the function directly. That is:

```
def foo(fn):
  ...

foo(torch.igamma)
foo(torch.igammac)
```